service/controller/resources/vaultcrt/resource.go

```diff
@@ -28,6 +28,8 @@ const (
 	// UpdateTimestampLayout is the time layout used to format and parse the
 	// update timestamps tracked in the annotations of the Kubernetes secrets.
 	UpdateTimestampLayout = "2006-01-02T15:04:05.000000Z"
+	// managedByLabel is used to flag resources create by this operator.
+	managedByLabel = "giantswarm.io/managed-by"
```

Comment:
For consistency, would it make sense to move this to `label` package?

Reply:
make sense, done.